QUEST_LV_0100.tsv

```diff
@@ -8606,58 +8606,58 @@ QUEST_LV_0100_20151001_008605	$Since we were able to sealed the Kartas Hunter he
-QUEST_LV_0100_20151001_008629	Right now... I can hardly move due to the injury.{nl}But, if I stay like this, the chasers would find out about it.
-QUEST_LV_0100_20151001_008630	I think we should move.{nl}Can you lure the chasers so that I could go to Guoda Empty Lot?{nl}
-QUEST_LV_0100_20151001_008631	When we fire this disposable signal bomb, they will follow you.{nl}I am counting on you...
+QUEST_LV_0100_20151001_008625	I can tell you this at least.{nl}The keepsakes of Lydia Schaffen...{nl}I came out from the Astral Tower to protect them with my colleagues.{nl}
+QUEST_LV_0100_20151001_008626	The preceding leader suspiciously died and Ignas became the next leader of the Astral Tower.{nl}Then, his tyranny started.{nl}
+QUEST_LV_0100_20151001_008627	He told everyone that the old rules of Lydia Schaffen should be abandoned and that we should start over...{nl}He said to eliminate everything that was related to Lydia Schaffen.{nl}
+QUEST_LV_0100_20151001_008628	But Lydia Schaffen was everything the Astral Tower was about, she was the symbolic figure of all archers.{nl}After we regained consciousness, we ran away with our colleagues and the keepsakes.{nl}
+QUEST_LV_0100_20151001_008629	Right now... I can hardly move due to my injury...{nl}But if I stay like this, the chasers will find out about me.
+QUEST_LV_0100_20151001_008630	I think we should move.{nl}Can you lure the chasers so I can go to the Guoda Empty Lot?{nl}
+QUEST_LV_0100_20151001_008631	When we fire this disposable signal bomb, they'll follow you.{nl}I'm counting on you...
 QUEST_LV_0100_20151001_008632	As you lure them, I will move secretly.{nl}Let's meet at Guoda Empty Lot.
 QUEST_LV_0100_20151001_008633	We were able to move safely due to your help.{nl}You seem like a good person so please receive this.{nl}
 QUEST_LV_0100_20151001_008634	This is the book recorded about the Roxona Conflict.{nl}I am not sure whether you are interested, but please receive it as a token of my gratitude.
-QUEST_LV_0100_20151001_008635	Ah, I came up with a good idea as the chasers ran towards the signal bomb you fired.{nl}I am going to make a trap by attaching signal bomb onto a bomb.{nl}
-QUEST_LV_0100_20151001_008636	Then my colleagues would know that I am here...{nl}Maybe, we could also lure the demons.{nl}
-QUEST_LV_0100_20151001_008637	To do so, we need to something to make smoke...{nl}But, I've used everything I've had.{nl}
-QUEST_LV_0100_20151001_008638	Good. I saw Red Leaves as I ran away. To Ruda Trail. {nl}Can you collect those?
+QUEST_LV_0100_20151001_008635	Ah, I came up with a good idea when the chasers ran towards the signal bomb you fired.{nl}I'm going to make a trap by attaching signal bomb onto a bomb.{nl}
+QUEST_LV_0100_20151001_008636	Then my colleagues will know that I'm here...{nl}Maybe we can also lure the demons.{nl}
```

Comment:
That way, my colleagues will know that I'm here...{nl}Maybe it will also lure the demons.{nl}

assuming line 8635 is the preceding line